test/modules/TestResultCoordinator/Startup.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator
+{
+    using Microsoft.AspNetCore.Builder;
+    using Microsoft.AspNetCore.Hosting;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.DependencyInjection;
+
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            this.Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddMvc();
+        }
+
+        // TODO: Remove warning disable for Obsolete when project is moved to dotnetcore 3.0
+#pragma warning disable 612, 618
```

Comment:
Why do you need this?

Reply:
This is also in the TestAnalyzer [here](https://github.com/ancaantochi/iotedge/blob/ancan/network-controller/edge-modules/TestAnalyzer/Startup.cs#L25). Looked at the blame and it was taken from this [PR to update dotnet core to 3.0 for arm64](https://github.com/ancaantochi/iotedge/commit/ad345efae692bbf3e28dc3d763f32ab25d667265).